deployment/ansible/roles/nyc-trees.tiler/meta/main.yml

```diff
@@ -0,0 +1,6 @@
+---
+dependencies:
+  - { role: "azavea.nginx", nginx_delete_default_site: True }
+  - { role: "azavea.mapnik" }
+  - { role: "azavea.daemontools" }
+  - { role: "azavea.nodejs" }
```

Comment:
`g++` is missing during the Node.js dependency installation process. Need to add the `azavea.build-essential` role.